test/e2e/network/network_policy.go

```diff
@@ -574,6 +816,30 @@ func createNetworkClientPod(f *framework.Framework, namespace *v1.Namespace, pod
 	return pod
 }
 
+// Patch client pod with a map value
+func updateNetworkClientPodLabel(f *framework.Framework, namespace *v1.Namespace, podName string, patchOperation string, patchPath string, patchValue map[string]string) *v1.Pod {
+	type patchMapValue struct {
+		Op    string            `json:"op"`
+		Path  string            `json:"path"`
+		Value map[string]string `json:"value"`
+	}
+	payload := []patchMapValue{{
+		Op:    patchOperation,
+		Path:  patchPath,
+		Value: patchValue,
+	}}
+	payloadBytes, err := json.Marshal(payload)
+	framework.ExpectNoError(err)
+
+	pod, err := f.ClientSet.
+		CoreV1().
+		Pods(namespace.Name).
+		Patch(podName, types.JSONPatchType, payloadBytes)
```

Comment:
nit: Why we need to separate this single API call into multiple lines? Just curious because I never see this pattern on a single API call.

Reply:
Just followed the pattern I've seen in another file. But, it can avoid having a big line, which is not actually the case in here, so I've updated to a single line.